docs/pages/docs/workflows/typescript.mdx

```diff
@@ -2,6 +2,8 @@ import Callout from 'components/Callout';
 
 # TypeScript integration
 
+## Strict typing of messages
+
 `next-intl` integrates with TypeScript out-of-the box without additional setup. You can however provide the shape of your messages to get autocompletion and type safety for your namespaces and message keys.
```

Comment:
I think this sentence could be split into a generic sentence right below the main headline and then one specific to messages here. What do you think?